com.unity.netcode.gameobjects/Runtime/Metrics/NetworkObjectProvider.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Unity.Netcode
 {
-    class NetworkObjectProvider : INetworkObjectProvider
+    internal class NetworkObjectProvider : INetworkObjectProvider
```

Comment:
I can't help but ask this again - are we ever going to support unity's coding standards? Practically everywhere else that c# code is written, you're not supposed to add extraneous access modifiers. (Not blocking this PR with this question ofc)